### PR TITLE
Fix file sizes of LLM's

### LINKED ISSUE #910: 🐛 [Bug]: CodeLlama-7B instruct download stuck
### Bug description

CodeLlama-7B instruct download stuck at 93.4%. I am able to use the model though. This doesn't happen with other models like Mistral-7B-OpenOrca.

### Steps to reproduce

Download CodeLlama-7B-model

### Environment Information

Docker version 24.0.7, build afdd53b

### Screenshots

image

### Relevant log output

```shell
INFO:     192.168.29.143:50268 - "GET /model/CodeLlama-7B-Instruct/download/status HTTP/1.1" 200 OK
INFO:     192.168.29.143:50268 - "GET /model/CodeLlama-7B-Instruct/download/status HTTP/1.1" 200 OK
INFO:     192.168.29.143:50268 - "GET /model/CodeLlama-7B-Instruct/download/status HTTP/1.1" 200 OK
INFO:     192.168.29.143:50268 - "GET /model/CodeLlama-7B-Instruct/download/status HTTP/1.1" 200 OK
INFO:     192.168.29.143:50268 - "GET /model/CodeLlama-7B-Instruct/download/status HTTP/1.1" 200 OK
INFO:     192.168.29.143:50268 - "GET /model/CodeLlama-7B-Instruct/download/status HTTP/1.1" 200 OK
INFO:     192.168.29.143:50268 - "GET /model/CodeLlama-7B-Instruct/download/status HTTP/1.1" 200 OK
INFO:     192.168.29.143:50268 - "GET /model/CodeLlama-7B-Instruct/download/status HTTP/1.1" 200 OK
INFO:     192.168.29.143:50268 - "GET /model/CodeLlama-7B-Instruct/download/status HTTP/1.1" 200 OK
INFO:     192.168.29.143:50268 - "GET /model/CodeLlama-7B-Instruct/download/status HTTP/1.1" 200 OK
INFO:     192.168.29.143:50268 - "GET /model/CodeLlama-7B-Instruct/download/status HTTP/1.1" 200 OK
INFO:     192.168.29.143:50268 - "GET /model/CodeLlama-7B-Instruct/download/status HTTP/1.1" 200 OK
INFO:     192.168.29.143:50268 - "GET /model/CodeLlama-7B-Instruct/download/status HTTP/1.1" 200 OK
INFO:     192.168.29.143:50268 - "GET /model/CodeLlama-7B-Instruct/download/status HTTP/1.1" 200 OK
INFO:     192.168.29.143:50268 - "GET /model/CodeLlama-7B-Instruct/download/status HTTP/1.1" 200 OK
INFO:     192.168.29.143:50268 - "GET /model/CodeLlama-7B-Instruct/download/status HTTP/1.1" 200 OK
INFO:     192.168.29.143:50268 - "GET /model/CodeLlama-7B-Instruct/download/status HTTP/1.1" 200 OK
```

### Confirmations

- [X] I'm running the latest version of the main branch.
- [X] I checked existing issues to see if this has already been described.

### PR DESCRIPTION
**Pull request by Smartappli**
Reorganization of models.json file

Fixes #910

### CODE REVIEW

**Comment by Smartappli** [comment]
Work in progress....

**Comment by Smartappli** [comment]
Models Filesize control
-------------------------
Alfred-40B
Falcon-7B
Falcon-7B Instruct
Falcon-40B
Falcon-40B Instruct
CodeLlama-7B                                            OK
CodeLlama-7B-Instruct                              OK
CodeLlama-13B-Instruct                            OK

**Comment by gaby** [comment]
I'm not sure what are you changing? @Smartappli

**Comment by Smartappli** [comment]
> I'm not sure what are you changing? @Smartappli

I'm testing each module one after the other to be sure

**Comment by gaby** [comment]
@Smartappli You can do that with the huggingface api.  I just need to update the sizes. Some of the them are off

**Comment by gaby** [comment]
Try this:

```python
from huggingface_hub import list_files_info

print(list(list_files_info(repo_id="TheBloke/CodeLlama-7B-GGUF",
        repo_type="model",
        revision="main",
    ))
)
```

It should print the size

**Comment by Smartappli** [comment]
> You can do that with the huggingface api. I just need to update the sizes. Some of the them are off

we need to update the size of the ones mentioned KO :)

**Comment by Smartappli** [comment]
Wow, they all work:)

**Comment by gaby** [comment]
@Smartappli Thanks for the help, will review later today. 💪

**Comment by Smartappli** [comment]
image

**Comment by gaby** [comment]
Will do, thanks

File: README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Instructions for setting up Serge on Kubernetes can be found in the [wiki](https
 | **Alfred** | 40B |
 | **CodeLLaMA** | 7B, 13B |
 | **Falcon** | 7B, 7B-Instruct, 40B, 40B-Instruct |
-| **LLaMA 2**  | 7B, 13B, 70B, 70B-Chat, 70B-OASST |
```
Comment (gaby) [inline README.md]: Why? `-chat` has more downloads than `oasst`

Comment (Smartappli) [reply README.md]: Sorry for the mistake

**Pull request merged**